test/puppetlabs/trapperkeeper/authorization/testutils.clj

```diff
@@ -2,9 +2,12 @@
   (:require [puppetlabs.ssl-utils.simple :as ssl]))
 
 (defn request
-  "Builds a ring request"
-  [path method certificate ip]
-  {:uri path :request-method method :remote-addr ip :ssl-client-cert certificate})
+  "Build a ring request for testing"
+  ([path method] (request path method "127.0.0.1"))
+  ([path method ip] {:uri path :request-method method :remote-addr ip})
```

Comment:
Yeah, although I thought we would use the various "add something to the request" helpers rather than hardcoding the keys and assoc'ing them in ourselves; that was the point of doing this, right?